.circleci/config.yml

```diff
@@ -7,10 +7,6 @@
 #     - ems: Emscripten
 version: 2.1
 parameters:
-  ubuntu-1804-docker-image:
-    type: string
-    # solbuildpackpusher/solidity-buildpack-deps:ubuntu1804-4
```

Comment:
Can the buildpack be removed then or is it still needed? If we remove it, better to just create an issue for it and do it separately.

Reply:
Oh you actually removed it. Does that retrigger any builds?

Reply:
You mean delete it in the docker account? Is there need for that? May be nice to have them around there, if we ever want to reproduce historic compiler builds - and I did remove updating the buildpack in this PR (see the github action job changes), because for anything but historic builds, we won't need them anymore, so we don't need to keep them updated.

Reply:
> You mean delete it in the docker account?

No, I meant delete it in our scripts.

Reply:
That's done in the PR, though! Do you think it's a problem to do it right here? As far as I'm concerned it belongs in this PR, but I'd be happy to move it out, if you think that's better...

Reply:
> Oh you actually removed it. Does that retrigger any builds?

Somehow github swallowed that comment earlier... yes, the script is removed here and also the github action trigger, so this will not trigger any rebuilds.